precompiles/slinky/abi.json

```diff
@@ -1,90 +1,67 @@
 {
-	"data": {
-		"bytecode": {
-			"functionDebugData": {},
-			"generatedSources": [],
-			"linkReferences": {},
-			"object": "",
-			"opcodes": "",
-			"sourceMap": ""
-		},
-		"deployedBytecode": {
-			"functionDebugData": {},
-			"generatedSources": [],
-			"immutableReferences": {},
-			"linkReferences": {},
-			"object": "",
-			"opcodes": "",
-			"sourceMap": ""
-		},
-		"gasEstimates": null,
-		"methodIdentifiers": {
-			"getPrice(string,string)": "3d0f34da"
-		}
-	},
-	"abi": [
-		{
-			"inputs": [
-				{
-					"internalType": "string",
-					"name": "base",
-					"type": "string"
-				},
-				{
-					"internalType": "string",
-					"name": "quote",
-					"type": "string"
-				}
-			],
-			"name": "getPrice",
-			"outputs": [
-				{
-					"components": [
-						{
-							"internalType": "uint64",
-							"name": "id",
-							"type": "uint64"
-						},
-						{
-							"internalType": "uint64",
-							"name": "nonce",
-							"type": "uint64"
-						},
-						{
-							"internalType": "uint64",
-							"name": "decimals",
-							"type": "uint64"
-						},
-						{
-							"components": [
-								{
-									"internalType": "uint64",
-									"name": "blockHeight",
-									"type": "uint64"
-								},
-								{
-									"internalType": "uint256",
-									"name": "blockTimestamp",
-									"type": "uint256"
-								},
-								{
-									"internalType": "uint256",
-									"name": "price",
-									"type": "uint256"
-								}
-							],
-							"internalType": "struct QuotePrice",
-							"name": "price",
-							"type": "tuple"
-						}
-					],
-					"internalType": "struct GetPriceResponse",
-					"name": "response",
-					"type": "tuple"
-				}
-			],
-			"stateMutability": "view",
-			"type": "function"
-		}
-	]
-}
\ No newline at end of file
+  "abi": [
```

Comment:
it's very unfortunate to have this file marked as modified, since if i understand correctly you didn't really modified it, but regenerating it produced the same json with its field ordered in a different way, right?

Reply:
if that's the case I'd like to introduce some post-generation step that sorts the json deterministically, so I don't have to go through this file to check if something was changed or not 😅  (but that's to be done in a separate pr)

Reply:
Yeah, it's essentially the same, just without unneeded "data" object (we do not use it in any way, only "abi" object) in the beginning. But I can revert it for now so it would make this PR clearer

Reply:
Reverted